dataactbroker/handlers/fileHandler.py

```diff
@@ -58,7 +59,9 @@ class FileHandler:
     STATUS_MAP = {"waiting":"invalid", "ready":"invalid", "running":"waiting", "finished":"finished", "invalid":"failed", "failed":"failed"}
     VALIDATION_STATUS_MAP = {"waiting":"waiting", "ready":"waiting", "running":"waiting", "finished":"finished", "failed":"failed", "invalid":"failed"}
 
-    def __init__(self,request,interfaces = None,isLocal= False,serverPath =""):
+    UploadFile = namedtuple('UploadFile', ['file_type', 'upload_name', 'file_name', 'upload_job_id'])
```

Comment:
It doesn't look like `upload_job_id` is ever used. Do we need it for something? 10 points for using a named tuple here.